R/mrgindata.R

```diff
@@ -81,6 +81,18 @@ convert_character_cmt <- function(data, mod) {
   return(data)
 }
 
+signal_drop <- function(dm, x, to_signal, context) {
+  drop <- setdiff(names(x), dimnames(dm)[[2]])
+  drop <- intersect(drop, to_signal)
+  for(d in drop) {
+    type <- paste0(class(x[[d]]), collapse = ",")
+    msg <- c(context, " dropped column: ", d, " (", type, ")")
+    msg <- paste(msg)
```

Comment:
Hmm, what's this `paste` doing?  Won't the result be the same as the input?


```
> x <- c("foo", "bar")
> identical(x, paste(x))
[1] TRUE
```

Reply:
I see ... I think I was intending to collapse that vector but message is doing it for me. Thanks.